device/isis/routerbasic.yaml

```diff
@@ -0,0 +1,20 @@
+components:
+  schemas:
+    Isis.Basic:
+      description: |-
+        This contains ISIS router basic properties.
+      type: object
+      properties:
+        ipv4_te_router_id:
+          description: >-
+            IPv4 Traffic Engineering(TE) router id.
+            This address should be configured as an IPv4 Loopback address in 'ipv4_loopbacks'
+            in the Device.      
+          type: string
+          format: ipv4
+          default: "1.1.1.1"
+        hostname:
+          description: >-
+            Host name for the router. The host name is transmitted in all the packets sent from the router.
+          type: string
+          default: ixia
```

Comment:
default shouldn't be `ixia`

Reply:
to expand on this, if indeed this field is necessary, then please mark it as required. otherwise we should resort to using a different name as default.

Reply:
this is not mandatory and hence default attribute is removed now.